src/ExpressionEvaluator/Core/Source/ExpressionCompiler/EvaluationContextBase.cs

```diff
@@ -17,6 +17,7 @@ internal abstract class EvaluationContextBase
         internal static readonly AssemblyIdentity SystemCoreIdentity = new AssemblyIdentity("System.Core");
         internal static readonly AssemblyIdentity SystemXmlIdentity = new AssemblyIdentity("System.Xml");
         internal static readonly AssemblyIdentity SystemXmlLinqIdentity = new AssemblyIdentity("System.Xml.Linq");
+        internal static readonly AssemblyIdentity MicrosoftVisualBasicIdentity = new AssemblyIdentity("Microsoft.VisualBasic");
```

Comment:
Unless you feel strongly, I'd prefer to keep them together for discoverability (i.e. so we don't create duplicate constants because we don't know about existing ones).